submarine-commons/commons-utils/src/main/java/org/apache/submarine/commons/utils/SubmarineConfiguration.java

```diff
@@ -337,8 +337,8 @@ public String getWebsocketMaxTextMessageSize() {
    * @param name the submitter's name
    * @return class name
    */
-  public String getSubmitterEntry(String name) {
-    return getStringValue(String.format(ConfVars.SUBMARINE_SUBMITTERS_ENTRY.getVarName(), name), "");
+  public String getSubmitterEntryClass(String name) {
```

Comment:
Change `getSubmitterEntryClass` to `getSubmitterClass` ?

Reply:
Good catch, thanks for your review. Done.